python/ray/node.py

```diff
@@ -400,6 +402,16 @@ def _get_unused_port(self):
         s.bind(("", 0))
         port = s.getsockname()[1]
         s.close()
+
+        # Try to generate a port that is far above the 'next available' one
```

Comment:
can you add some justification here? The PR description does a good job, maybe link to the issue as well?

Reply:
Will Do